.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.8.7
+  - 1.17.5
```

Comment:
I guess your can simply remove the travis file.

Reply:
I'll just leave it unchanged ;)

Reply:
On a second thought, yes, probably should just remove it and avoid confusion!